app/controllers/users/registrations_controller.rb

```diff
@@ -4,7 +4,13 @@
 class Users::RegistrationsController < Devise::RegistrationsController
   # POST /resource
   def create
+    if User.where(:email => params[:user][:email]).any?
+      flash[:warning] = "Looks like that email is already taken you can login here or <a href=#{new_user_registration_path}>go back to sign up</a>".html_safe
+      redirect_to new_user_session_path(user: {email: params[:user][:email]}) and return
```

Comment:
Can you change the `and` to `&&` and add spaces inside the brackets (i.e., `{ email.... email] }`)?

Reply:
Hey @rlgreen91 because of how precedence works using `&&` here would break the functionality - there's a pretty good explanation of that here: https://stackoverflow.com/questions/39629976/ruby-return-vs-and-return